src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  background-color: #f9f9f9;
+}
+
+.main-header {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  border-bottom: 1px solid #ddd;
+  position: relative;
+}
```

Comment:
Don't fix container sizes like `width` and `height` unless there is a specific requirement for that. Let the content size dictate the container's size to avoid overflow or accidental scroll bars.